.docker/Dockerfile-CPU-Base

```diff
@@ -3,7 +3,7 @@
 # ------------------------------------------------------------------
 # Ubuntu           18.04
 # OpenMPI          latest       (apt)
-# cmake            3.10         (git)
+# cmake            3.18         (git)
```

Comment:
3.18 is too new. Ubuntu 18.04 supports 3.10
https://launchpad.net/ubuntu/bionic/+source/cmake
Ubuntu 20.04 supports 3.16
https://launchpad.net/ubuntu/focal/+source/cmake
I think that we don't want to break the builds for users on these platforms.

Reply:
[Snapcraft](https://snapcraft.io/cmake) which is installed by default in Ubuntu has CMake 3.18.4 as default and is super easy to install:

```sh
snap install --classic cmake
```

And even so, you could download the [compiled version for Linux ](https://github.com/Kitware/CMake/releases/download/v3.18.4/cmake-3.18.4-Linux-x86_64.tar.gz) from their repo and it works out of the box. And installing from source is also super easy.

We could try to run it with CMake 3.16, but I think the minimum version is 3.15 although I haven't tested.